packages/database/src/migrations/20210902055844-UpdateUNFPASbRawDataDownloadSurvey-modifies-data.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+import { deleteObject, insertObject, generateId, updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const OLD_REPORT_CODE = 'UNFPA_Raw_Data_Reproductive_Health_Facility';
+const NEW_REPORT_CODE = 'UNFPA_Raw_Data_Reproductive_Health_Facility_SB';
+const DASHBOARD_CODE = 'SB_UNFPA_Raw Data Downloads';
+
+const buildSurveyExportConfig = ({ surveyCode, surveyName }) => ({
+  surveys: [{ code: surveyCode, name: surveyName }],
+  exportDataBuilder: {
+    dataBuilder: 'rawDataValues',
+    dataBuilderConfig: {
+      skipHeader: false,
+      surveysConfig: {
+        [surveyCode]: { entityAggregation: { dataSourceEntityType: 'facility' } },
+      },
+      transformations: [{ type: 'transposeMatrix' }],
+    },
+  },
+});
+
+const swapDashboardItemInRelation = async (db, dashboardCode, oldItemCode, newItemCode) =>
+  db.runSql(`
+    UPDATE dashboard_relation
+    SET child_id = new_di.id
+    FROM dashboard d, dashboard_item old_di, dashboard_item new_di
```

Comment:
Huh... I never knew you could update records from the result set of a join! Nice one!!